integration/harness-e2e-cli/package.json

```diff
@@ -44,7 +44,7 @@
     "karma-jasmine-html-reporter": "~1.7.0",
     "selenium-webdriver": "3.6.0",
     "ts-node": "~10.7.0",
-    "typescript": "file:../../node_modules/typescript",
+    "typescript": "~4.7.2",
```

Comment:
I had to lock down the TS version of all of these integration tests, because the CLI hasn't been updated to handle 4.8 yet.